RedemptionQuick/sets/carddata.txt

```diff
@@ -49,7 +49,7 @@ Abner's Spear (Wa)	Warriors	Abner's_Spear_(Wa)	EE	Brown	2	4				Common	II Samuel
 Abomination of Desolation	Rock of Ages	Abomination_of_Desolation_(RA)	EE	Black				False Religious Practice, Limit of one in Territory	If used by a Greek, place in opponent's territory. Each time opponent draws cards (except during draw phase), you may discard a card in that territory except a Lost Soul.	Fixed	Daniel 11:31	
 Abraham's Descendant	Disciples	Abraham's_Descendant_(Di)	GE	Blue	3	3	Territory	Prophesy	Interrupt a Curse or Evil Enhancement in play and add it to your hand.  Search each discard pile for a good Enhancement and place them beneath owners' decks.	Fixed	Genesis 12:3	
 Abraham's Servant to Ur	Patriarchs	Abraham's_Servant_to_Ur_(Pa)	GE	Blue	2	3			All special abilities except banding on characters and enhancements, except this one, are negated.  Battle is determined by the numbers.	Rare	Genesis 24:10	
-Abram/Abraham	Patriarchs	Abram_Abraham_(Pa)	Hero	Blue	10-May	11-Jun			Hero starts as Abram.  After he makes a successful rescue, he becomes Abraham and can interrupt and prevent all special abilities on all Lost Soul cards except the Proverbs 22:14 Lost Souls card.	Ultra Rare	Genesis 17:4-5	
+Abram/Abraham	Patriarchs	Abram_Abraham_(Pa)	Hero	Blue	10	11			Hero starts as Abram.  After he makes a successful rescue, he becomes Abraham and can interrupt and prevent all special abilities on all Lost Soul cards except the Proverbs 22:14 Lost Souls card.	Ultra Rare	Genesis 17:4-5	
```

Comment:
Abram/Abraham should have strength 5/10 and toughness 6/11

Reply:
Corrected